docs/source/tutorials/basics/basics_setup.rst

```diff
@@ -95,7 +95,8 @@ tutorial:
 
 .. literalinclude:: code/hpo_main.py
    :caption: hpo_main.py: Configuration space
-   :lines: 25-37
+   :start-at: from syne_tune.config_space import randint, uniform, loguniform
```

Comment:
A risk would be if we import more things on that line

Reply:
But I think it's fine given that it's docs code: https://github.com/awslabs/syne-tune/blob/main/docs/source/tutorials/basics/code/hpo_main.py, so if people change it, they will reasonably expect the docs to change, and reviewers should check for this.